.pre-commit-config.yaml

```diff
@@ -1,13 +0,0 @@
-repos:
-- repo: https://github.com/pre-commit/pre-commit-hooks
-  rev: v4.3.0
-  hooks:
-    - id: check-ast
-    - id: check-yaml
-    - id: mixed-line-ending
-    - id: trailing-whitespace
-- repo: https://github.com/pycqa/isort
-  rev: 5.10.1
-  hooks:
-    - id: isort
-      args: [--line-length=120]
```

Comment:
Even if the CI/CD configuration is being removed I don't see why the pre-commit configuration itself needs to be removed.